.github/workflows/c3-e2e-dependabot.yml

```diff
@@ -3,9 +3,9 @@
 name: C3 E2E Tests (Dependabot)
 
 on:
-  pull_request:
+  pull_request_target:
     paths:
-      - packages/create-cloudflare/**
+      - .changeset/c3-frameworks-update-*.md
```

Comment:
This is so that this workflow only runs after the changeset one was generated (since before that running these tests doesn't really make sense and it would also require more complex logic in the `git diff` command (as it could run on top of one or two commits so we'd need to handle both cases somehow)